Firmware/Marlin_main.cpp

```diff
@@ -3575,6 +3575,10 @@ void gcode_M701(float fastLoadLength, uint8_t mmuSlotIndex){
         current_position[E_AXIS] += fastLoadLength;
         plan_buffer_line_curposXYZE(FILAMENTCHANGE_EFEED_FIRST); //fast sequence
 
+        if (eFilamentAction == FilamentAction::AutoLoad || raise_z_axis) { // backwards compatibility for 3.12 and older FW
+          raise_z_above(MIN_Z_FOR_LOAD);
```

Comment:
I'm not sure if this is the right place for `raise_z_above(`) since the M701 gcode has a Z parameter now, can we use it instead?

```
#### Usage

        M701 [ P | T | L | Z ]

    #### Parameters
    - `P` - n index of MMU slot (zero based, so 0-4 like T0 and T4)
    - `T` - Alias of `P`. Used for compatibility with Marlin
    - `L` - Extrude distance for insertion (positive value)(manual reload)
    - `Z` - Move the Z axis by this distance. Default value is 0 to maintain backwards compatibility with older gcodes.
    */
```

Reply:
Z parameter in M701 uses raise_z not raise_z_above wich behaves different.
It also moves the head too early because M701 is used when the filament sensor is triggered and temp is >= target temp.